docs/ref/models/expressions.txt

```diff
@@ -493,6 +502,11 @@ The ``filter`` argument takes a :class:`Q object <django.db.models.Q>` that's
 used to filter the rows that are aggregated. See :ref:`conditional-aggregation`
 and :ref:`filtering-on-annotations` for example usage.
 
+The ``order_by`` argument takes a field name (with an optional ``"-"`` prefix
+which indicates descending order) or an expression (or a tuple or list of
+strings and/or expressions) that specifies the ordering of the elements in the
+result.
```

Comment:
I couldn't spot tests around a tuple/list of fields - can you check/add tests for these?

I also wonder if we can say that this is similar to what you can input to `order_by()`

Reply:
> I also wonder if we can say that this is similar to what you can input to order_by()

It should effectively accept similar inputs.

Reply:
I have updated the documentation to include something about this. Let me know if you would like any changes.